dev/com.ibm.ws.jaxrs.2.1.appSecurity/bnd.bnd

```diff
@@ -0,0 +1,42 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.jaxrs.2.1.appSecurity
+
+-dsannotations: \
+  com.ibm.ws.jaxrs21.providers.security.SecurityAnnoProviderRegister
+
+Export-Package: \
+  com.ibm.ws.jaxrs21.security;thread-context=true
```

Comment:
Do we really need to export it if it is just returned from SecurityAnnoProviderRegister?